services/net/contentmigration/migrators/PaperMigrator.cs

```diff
@@ -49,8 +50,8 @@ public override SourceContent CreateSourceContent(LookupModel lookups, SourceMod
         Logger.LogDebug("NewItem.RSN: {rsn}, PublishedDateTime: {publishedDateTime}, ToUtc: {publishedDateTimeInUtc}", newsItem.RSN, publishedOnInDefaultTimeZone, publishedOnInUtc);
 
         var newsItemTitle = newsItem.GetTitle();
-        var sanitizedSummary = SanitizeLineBreaks(newsItem.Summary);
-        var sanitizedBody = SanitizeLineBreaks(newsItem.Text);
+        var sanitizedSummary = TNO.Core.Extensions.StringExtensions.ConvertTextToParagraphs(newsItem.Summary, @"\r\n?|\n|\|");
```

Comment:
One thing I like about extension methods is how they can be called.
`var sanitizedSummary = newsItem.Summary.ConvertTextToParagraph(@"\r\n?|\n|\|");`

Reply:
funnily enough I always forget about using extensions for simple types...